wiki/Writing-tests-with-good-behavioural-coverage.md

```diff
@@ -26,20 +31,20 @@ Focusing on behavioral coverage is essential because:
 
 Let's understand it better with a sample function. Consider this function that validates a name:
 
-```
+```kotlin
 fun getName(name: String? = " ") =
-    name?.takeIf { it.all { char -> char.isLetterOrWhitespace() } }
-        ?: throw IllegalArgumentException("Invalid name")
+  name?.takeIf { it.all { char -> char.isLetterOrWhitespace() } }
+    ?: throw IllegalArgumentException("Invalid name")
 ```
 
 ### Line Coverage Testing
 
 A basic test case for line coverage might look like this:
 
-```
+```kotlin
 @Test fun testValidName() {
-    // Tests line coverage by hitting the line where name is accessed
-    assertThat(getName("Alice")).isEqualTo("Alice") 
+  // Tests line coverage by hitting the line where name is accessed
+  assertThat(getName("Alice"), equalTo("Alice")) 
```

Comment:
Yeah, I was moving things in and out with GitHub Markdown formatting and Android Studio. I should have accidently moved in an older version (will also cross check other updates I did since then), but I’ve now changed, `equalTo` to use `.isEqualTo`. Thanks!